src/renderer/src/components/ContextMenu.tsx

```diff
@@ -0,0 +1,194 @@
+      className
+    )}
+    checked={checked}
+    {...props}
+  >
+    <span className="absolute left-2 flex h-3.5 w-3.5 items-center justify-center">
+      <ContextMenuPrimitive.ItemIndicator>
+        <TbCheck aria-hidden={true} className="h-4 w-4" />
+      </ContextMenuPrimitive.ItemIndicator>
+    </span>
+
+    {children}
+  </ContextMenuPrimitive.CheckboxItem>
+))
+
+CheckboxItem.displayName = ContextMenuPrimitive.CheckboxItem.displayName
+
+const RadioItem = forwardRef<
+  ElementRef<typeof ContextMenuPrimitive.RadioItem>,
+  ComponentPropsWithoutRef<typeof ContextMenuPrimitive.RadioItem>
+>(({ className, children, ...props }, ref) => (
+  <ContextMenuPrimitive.RadioItem
+    ref={ref}
+    className={StyleHelper.mergeStyles(
+      'relative text-white transition-colors hover:bg-neon/10 focus:bg-neon/10 flex cursor-pointer select-none items-center rounded-sm py-1.5 pl-8 pr-2 text-xs outline-none data-[disabled]:cursor-default data-[disabled]:pointer-events-none data-[disabled]:opacity-50',
+      className
+    )}
+    {...props}
+  >
+    <span className="absolute left-2 flex h-3.5 w-3.5 items-center justify-center">
+      <ContextMenuPrimitive.ItemIndicator>
+        <TbCircleFilled aria-hidden={true} className="h-2 w-2 fill-current" />
+      </ContextMenuPrimitive.ItemIndicator>
+    </span>
+
+    {children}
+  </ContextMenuPrimitive.RadioItem>
+))
+
+RadioItem.displayName = ContextMenuPrimitive.RadioItem.displayName
+
+const Label = forwardRef<
+  ElementRef<typeof ContextMenuPrimitive.Label>,
+  ComponentPropsWithoutRef<typeof ContextMenuPrimitive.Label> & { inset?: boolean }
+>(({ className, inset, ...props }, ref) => (
+  <ContextMenuPrimitive.Label
+    ref={ref}
+    className={StyleHelper.mergeStyles('px-2 py-1.5 text-xs font-bold text-white', { 'pl-8': inset }, className)}
+    {...props}
+  />
+))
+
+Label.displayName = ContextMenuPrimitive.Label.displayName
+
+const Separator = forwardRef<
+  ElementRef<typeof ContextMenuPrimitive.Separator>,
+  ComponentPropsWithoutRef<typeof ContextMenuPrimitive.Separator>
+>(({ className, ...props }, ref) => (
+  <ContextMenuPrimitive.Separator
+    ref={ref}
+    className={StyleHelper.mergeStyles('-mx-1 my-1 h-px bg-gray-800', className)}
+    {...props}
+  />
+))
+
+Separator.displayName = ContextMenuPrimitive.Separator.displayName
+
+const Shortcut = ({ className, ...props }: React.HTMLAttributes<HTMLSpanElement>) => (
+  <span className={StyleHelper.mergeStyles('ml-auto text-1xs tracking-widest', className)} {...props} />
+)
+
+Shortcut.displayName = 'Shortcut'
+
+export default {
+  CheckboxItem,
+  Content,
+  Group,
+  Item,
+  Label,
+  Portal,
+  RadioGroup,
+  RadioItem,
+  Root,
+  Separator,
+  Shortcut,
+  Sub,
+  SubContent,
+  SubTrigger,
+  Trigger,
+}
```

Comment:
Use the composition pattern.
```suggestion
export const ContextMenu = {
  CheckboxItem,
  Content,
  Group,
  Item,
  Label,
  Portal,
  RadioGroup,
  RadioItem,
  Root,
  Separator,
  Shortcut,
  Sub,
  SubContent,
  SubTrigger,
  Trigger,
}
```